content/kapp/docs/latest/config.md

```diff
@@ -71,7 +71,8 @@ diffMaskRules:
 - `rebaseRules` (array) list of rebase rules
   - `path` (array) specifies location within a resource to rebase. Mutually exclusive with `paths`. Example: `[spec, clusterIP]`
   - `paths` (array of paths) specifies multiple locations within a resource to rebase. This is a convenience for specifying multiple rebase rules with only different paths. Mutually exclusive with `path`. Available in v0.27.0+.
```

Comment:
> - `path` (array)

path is an array of strings

> - `paths` (array of paths)

paths is an array of `path`s

Is there any way we can disambiguate this wording to ensure it isn't circular? i.e. I'm cautious that someone would read "paths is an array of paths" and misunderstand it to be recursive.